apps/epp-amend/fields/index.js

```diff
@@ -9,5 +9,17 @@ module.exports = {
       { type: 'maxlength', arguments: [16] },
       { type: 'minlength', arguments: [13] }
     ]
+  },
+  'amend-phone-number': {
+    mixin: 'input-text',
+    className: ['govuk-input'],
+    labelClassName: 'bold',
```

Comment:
It looks like the `labelClassName: 'bold'` is not respecting GDS style can you please use the GDS class `govuk-label--m ` this will format the `label` in compliance with the rest of gov.uk forms standards.
I leave this image as comparison
![image](https://github.com/user-attachments/assets/89d0f0f7-7e6b-4af7-a7b9-6ebba13d4595)

Reply:
thanks for the changes I will resolve this conversation